tools/ovc/openvino/tools/ovc/convert_impl.py

```diff
@@ -478,6 +478,12 @@ def _convert(cli_parser: argparse.ArgumentParser, args, python_api_used):
                     get_jax_decoder(args['input_model'], args)
                 else:
                     raise Error("JAX Frontend is not available.")
+            if model_framework == "tf":
+                try:
+                    from nncf.tensorflow.strip import strip
```

Comment:
```suggestion
                    from nncf.tensorflow.strip import strip as nncf_tf_strip
```

just avoid confusion and mix with built-in Python method